adam-core/src/main/scala/org/bdgenomics/adam/converters/VariantAnnotationConverter.scala

```diff
@@ -94,17 +94,17 @@ object VariantAnnotationConverter extends Serializable {
     AttrKey("omimId", attrAsString _, new VCFInfoHeaderLine("VAR", 1, VCFHeaderLineType.String, "MIM entry with variant mapped to rsID")))
 
   val INFO_KEYS: Seq[AttrKey] = Seq(
-    AttrKey("clippingRankSum", attrAsFloat _, new VCFInfoHeaderLine("ClippingRankSum", 1, VCFHeaderLineType.Float, "Z-score From Wilcoxon rank sum test of Alt vs. Ref number of hard clipped bases")),
-    AttrKey("readDepth", attrAsInt _, VCFStandardHeaderLines.getInfoLine(VCFConstants.DEPTH_KEY)),
+    // AttrKey("clippingRankSum", attrAsFloat _, new VCFInfoHeaderLine("ClippingRankSum", 1, VCFHeaderLineType.Float, "Z-score From Wilcoxon rank sum test of Alt vs. Ref number of hard clipped bases")),
```

Comment:
@erictu why did we drop these? Were these the `VariantCallingAnnotation` fields that we dropped?